### PR TITLE
Implemented and tested python-decouple lib

### PR DESCRIPTION
**Pull request by Riverfount**
#2

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
-*.py[cod]
 *$py.class
 
+# Setting up of PyCharm
+.idea/
```
Comment (renzon) [inline .gitignore]: I must keep IDE/Test editor stuff out of gitignore, use gitignore global localy for that.

File: .gitignore [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
+*.sqlite3
```
Comment (renzon) [inline .gitignore]: I am not so sure anyone would keep and kind of sqlite3 file out of source control. Maybe it's better also to include that on gitignore global locally

File: manage.py [hunk 3]
```diff
@@ -3,7 +3,9 @@
 import sys
 
 if __name__ == "__main__":
+
```
Comment (renzon) [inline manage.py]: Any specific reason to include these 2 blank lines?

File: .gitignore (hunk 1 above)
Comment (Riverfount) [reply .gitignore]: Ok! I understanding and I do it for the next PR!

File: .gitignore (hunk 2 above)
Comment (Riverfount) [reply .gitignore]: Ok! The same form of the first question I do it for the next PR!

File: manage.py (hunk 3 above)
Comment (Riverfount) [reply manage.py]: No, I think was a typo error!

**Comment by Riverfount** [comment]
What I need to do to fix the problems and my PR will be accepted?

**Comment by renzon** [comment]
Once we will compare django decouple with dynaconf, only one of those will get into the master branch. So we can keep each approach in a separate branch and merge only de chosen one later.

File: .gitignore (hunk 2 above)
Comment (renzon) [reply .gitignore]: In fact you don't need to raise another PR. You can just fix those in your repo and once you push the fix, this PR will be fixed as well ;)

**Comment by renzon** [comment]
As I mentioned, I've created branchs decouple and dynaconf so we can send each approach to respective branch and compare the code there. After this, on Tuesday's mtg we can decide each one to use. So I am closing this PR and you can create another one pointing to decouple branch.

**Pull request closed without merging**